fusor-ember-cli/app/controllers/review/summary.js

```diff
@@ -14,7 +14,10 @@ export default Ember.Controller.extend(NeedsDeploymentMixin, {
   undercloudUsername: 'admin',
   undercloudPassword: Ember.computed.alias("model.openstack_undercloud_password"),
 
-  undercloudUrl: Ember.computed('model.openstack_undercloud_ip_addr', function() {
```

Comment:
@jmontleon, @jwmatthews, not sure if this is a bug, but notice that `undercloudUrl` was using `openstack_undercloud_ip_addr` and not `openstack_undercloud_hostname`.